hack/create-s3-bucket.sh

```diff
@@ -6,28 +6,36 @@ BUCKET_NAME=$1
 
 REGION=$(aws configure get region)
 
+delete_bucket() {
+    aws s3 rb s3://"$BUCKET_NAME" --region "$REGION" --force  || true
+}
```

Comment:
I think this should just be moved to its own script.

Reply:
Fixed